src/components/pages/ItemList.js

```diff
@@ -0,0 +1,221 @@
+import React, { Component } from "react";
+import { Table, Card, Button, ButtonToolbar } from "react-bootstrap";
+import { Header } from "./Header";
+import Axios from "axios";
+import { MDBIcon, MDBBtn } from "mdbreact";
+
+export class ItemList extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      itemlist: [],
+      subtotal: 0,
+      items: [],
+      available_items: [],
+      show: false,
+      orderid: this.props.match.params.orderid,
+      additems: ["Denim", "Frock", "Tshirt", "Shirt", "Throuser", "Skirt"],
+      ordered_items: [],
+      itemid: "",
+      qty: "",
+      remain_items: []
+    };
+  }
+
+  componentDidMount() {
+    // const { orderid } = this.props.match.params;
+    // console.log(orderid);
+
+    Axios.get("http://localhost:5000/orders/")
+      .then(
+        res =>
+          this.setState({
+            itemlist: res.data
+          })
+        // console.log(res.data)
+      )
+      .catch(err => console.log(err));
+
+    Axios.get("http://localhost:5000/items/")
+      .then(
+        res =>
+          this.setState({
+            available_items: res.data
+          })
+        // console.log(res.data)
+      )
+      .catch(err => console.log(err));
+
+    Axios.get("http://localhost:5000/items/")
+      .then(
+        res =>
+          this.setState({
+            remain_items: res.data
+          })
+        // console.log(res.data)
+      )
+      .catch(err => console.log(err));
+  }
+
+  getItems() {
+    // console.log("**********************");
+    // console.log(this.state.itemlist);
+    // console.log(this.state.available_items);
```

Comment:
remove the unwanted code and make sure you use the correct logging method and message

Reply:
sure